libs/go-kibana-rest/docker-compose.yml

```diff
@@ -23,11 +23,14 @@ services:
       elasticsearch:
         condition: service_started
   kibana:
-    image: docker.elastic.co/kibana/kibana:8.14.1
+    image: docker.elastic.co/kibana/kibana:8.14.3
+    volumes:
+      - ./kibana.yml:/usr/share/kibana/config/kibana.yml
```

Comment:
IIUC everything in this config file could be supplied through the env, IMO it would be nicer to keep all the config here if possible.

Reply:
Good idea, I have added `XPACK_ENCRYPTEDSAVEDOBJECTS_ENCRYPTIONKEY` and removed this yaml.